test/lit.cfg

```diff
@@ -905,6 +909,16 @@ elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'wi
       config.target_sdk_name = "linux"
     config.target_runtime = "native"
     config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
+
+    config.libdispatch_build_dir = make_path(config.swift_obj_root, os.pardir,
```

Comment:
This is really unfortunate.  I know that I do not build the same way that the build-script sets things up (even ignoring that the paths are too long that way for Windows).  @davezarzycki may also be building differently.  Can you add an option to CMake that is wired down to the configuration instead?

Reply:
I am not sure which parts exactly I have to touch for this. Can you point me to an existing variable/configuration option where this is done?

Reply:
I recently wired libxml2 down.  Look at the CMakeList.txt in the test subdirectory.

Reply:
I am looking at [1] but I am still unsure how to tackle this. Note that the libdispatch that we are referencing here is not built as part of the Swift cmake build, but separately through build-script.
[1] https://github.com/apple/swift/commit/a37a18f16f7211238a50f8606e76dbaee9b60712

Reply:
We are now wiring the path through CMake!